src/menu/components/boxart.c

```diff
@@ -85,3 +112,26 @@ void component_boxart_draw (component_boxart_t *b) {
         );
     }
 }
+
+void component_boxart_draw_dd (component_boxart_t *b) {
```

Comment:
Same as above. I don't think we need another function for that. You can change existing function to have fixed positioning for lower right corner of the picture so it will look nice even with different boxart resolutions. As a bonus it would also support Japanese boxarts (these have portrait aspect ratio).